packages/core/src/routes/session.ts

```diff
@@ -229,6 +230,27 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     }
   );
 
+  router.get(
+    '/session/register/username/existence',
+    koaGuard({ body: object({ username: string() }) }),
+    async (ctx, next) => {
+      const { username } = ctx.guard.body;
+
+      assertThat(username, 'session.insufficient_info');
```

Comment:
`username` is always true?

Reply:
we have koaGuard already

Reply:
I was thinking about the case that 'username' can be an empty string which is not a desired situation.
Let's just put the checks of username validity together in koaGuard.